src/components/card/CardSection.tsx

```diff
@@ -35,6 +36,7 @@ export type CardSectionProps = ViewPropTypes & {
    * Will be used for the background when provided
    */
   imageSource?: ImageSourcePropType;
+  source?: ImageSourcePropType; //TODO: Remove after imageSource deprecation - should take it from Image props
```

Comment:
Not sure about the TODO, I think in this case `imageSource` will be removed and `source` will stay

Reply:
Please see my ansawer in CardImage

Reply:
Still, the prop `source` should still remain (as optional).

Reply:
Got you. Ok